A3A/addons/core/functions/Ammunition/fn_randomHandgun.sqf

```
@@ -0,0 +1,62 @@
+/*
```

I would recommend deleting this whole function.

The only time it's ever used is in initPetros, and if the handgun is just added as a fallback to randomRifle (see one of my other comments) then it won't be needed at all.